### DISCUSSION TITLE
✖ Not all operations have an unique name: post:

### DISCUSSION THREAD

**Discussion by AhmedSalam22** [dotansimha/graphql-code-generator#9756] [Q&A]
![image](https://github.com/dotansimha/graphql-code-generator/assets/35177448/97c8f084-3a21-4b6d-a8e6-169f8141acb3)

**Comment by timvandam** [comment]
I have a similar issue. I have two different files that have an operation with the same name, and graphql-codegen is complaining despite them being in different files. I'm also using import-types
